SWLOR.Game.Server/Service/Combat.cs

```diff
@@ -98,6 +101,7 @@ public static List<CombatDamageType> GetAllDamageTypes()
                 defenderDefense = 1;
 
             var statDelta = attackerStat - defenderStat;
+            if (deltaCap > 0) Math.Clamp(statDelta, -deltaCap, 8 + deltaCap);
```

Comment:
Does this cap need to be applied to the bible damage calc spreadsheet?

Reply:
Yep, there'll need to be several changes to the bible that I can pass you today